src/components/Interactive/Interactive.tsx

```diff
@@ -55,6 +72,22 @@ const SimpleSlider: React.FC<SimpleSliderProps> = React.memo(
 export const Interactive = () => {
     const [theme, setTheme] = useState<Theme>('dark');
     const [color, setColor] = useState<ColorTheme>(ColorTheme.Yellow);
+    const [autoplayDisabled, setAutoplayDisabled] = useState(false);
+
+    useEffect(() => {
+        if (typeof window === 'undefined') {
+            return;
+        }
+
+        // @ts-ignore
+        window.enableAutoplay = () => {
```

Comment:
How will it be used?
And the same question about `window.disableHand` / `window.enableHand`

Reply:
This is necessary for a test demonstration. It is planned to launch the functions from the developer console.
In the future, we will remove these functions.